parsers/ESIOS.py

```diff
@@ -24,6 +24,12 @@
     "ES->MA": "10209",
 }
 
+# Map each exchange to the needed factor to adjust from MWh to MW. Depends on the time granularity of the API for each request
+EXCHANGE_GRANULARITY_FACTOR_MAP = {
+    "AD->ES": 1,
+    "ES->MA": 4,
```

Comment:
Could you just add a comment that it's a 15min interval, so that when we look back at the code we understand this factor.